packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -111,6 +112,25 @@ const AutocompleteInner = <
   >,
   ref?: ForwardedRef<unknown>,
 ) => {
+  const getIsInputEmpty = (): boolean => {
```

Comment:
```suggestion
  const checkIsInputEmpty = (): boolean => {
```

https://kaluga-astral.github.io/style-guide/docs/rules/naming/functions